applications/poisson/sine/application.h

```diff
@@ -174,6 +174,11 @@ class Application : public ApplicationBase<dim, n_components, Number>
     this->param.multigrid_data.coarse_problem.preconditioner =
       MultigridCoarseGridPreconditioner::AMG;
     this->param.multigrid_data.coarse_problem.solver_data.rel_tol = 1.e-3;
+
+    AssertThrow(this->param.grid.create_coarse_triangulations ==
+                  this->param.multigrid_data.use_global_coarsening,
+                dealii::ExcMessage(
+                  "Can't use global_coarsening without activating create_coarse_triangulations"))
```

Comment:
This Assert is not placed optimally here in application. A user does not want to add this Assert to other test cases.